xds/src/main/java/io/grpc/xds/CdsLoadBalancer.java

```diff
@@ -125,124 +111,33 @@ public boolean canHandleEmptyAddressListFromNameResolution() {
   @Override
   public void shutdown() {
     logger.log(XdsLogLevel.INFO, "Shutdown");
-    switchingLoadBalancer.shutdown();
+    if (childLbState != null) {
+      childLbState.shutdown();
+    }
     if (xdsClientPool != null) {
       xdsClientPool.returnObject(xdsClient);
     }
   }
 
-  // TODO(sanjaypujare): remove once xDS security is released
-  private boolean isXdsSecurityEnabled() {
-    return enableXdsSecurity || Boolean.valueOf(System.getenv(XDS_SECURITY_ENV_VAR));
-  }
-
-  // TODO(sanjaypujare): remove once xDS security is released
-  @VisibleForTesting
-  void setXdsSecurity(boolean enable) {
-    enableXdsSecurity = enable;
-  }
-
```

Comment:
This is also unrelated style change. I also liked @sanjaypujare 's style. Maybe 
 ```java
return enableXdsSecurity || STATIC_VALUE
``` 
for `isXdsSecurityEnabled()` could improve it a bit. But the non-static `setXdsSecurity()` method for testing is better than modifing static value in test, in my opinion.

As pointed out by @ejona86 , we should not flip/flop unrelated lines in the PR. Just keeping as-is can make the diff of PR small. If you do want some improvement, make it a separate PR.

Reply:
Reading the environment variable should be done statically, and the field `enableXdsSecurity` is redundant.

The problem is, there could be so many such improvements (especially some tests completely doesn't make any sense), if you do them separately, there will be a lot of steps. Now that this refactoring is an involved internal structural change, it's more-or-less a rewrite.

Reply:
This code is going to go away once security is released so it's not worth so much discussion.